build/common.project.props

```
@@ -124,13 +124,17 @@
     <MinimumVisualStudioVersion>15.0</MinimumVisualStudioVersion>
     <VsixOutputDirName>VS15</VsixOutputDirName>
     <CodeAnalysisRuleSet Condition="'$(CodeAnalysisRuleSet)' == ''">$(BuildCommonDirectory)NuGet.ruleset</CodeAnalysisRuleSet>
-    <EnforceCodeStyleInBuild Condition=" '$(EnforceCodeStyleInBuild)' != '' " >true</EnforceCodeStyleInBuild>
+    <WarningsNotAsErrors>EnableGenerateDocumentationFile</WarningsNotAsErrors>
```

```suggestion
    <WarningsNotAsErrors>$(WarningsNotAsErrors);EnableGenerateDocumentationFile</WarningsNotAsErrors>
```

We should add not overwrite.

fwiw, I had no clue there's a warning identified as  `EnableGenerateDocumentationFile` and that it can be suppressed like that :D

---

I wasn't sure it would work tbh xD